metricbeat/docs/getting-started.asciidoc

```diff
@@ -0,0 +1,164 @@
+
+[float]
+[[install]]
+=== Step 1: Install {beatname_uc}
+
+Install {beatname_uc} as close as possible to the service you want to monitor.
+For example, if you have four servers with MySQL running, it's recommended that
+you run {beatname_uc} on each server. This allows {beatname_uc} to access your
+service from localhost and does not cause any additional network traffic or
+prevent {beatname_uc} from collecting metrics when there are network problems.
+Metrics from multiple {beatname_uc} instances will be combined on the
+Elasticsearch server.
+
+To download and install {beatname_uc}, use the commands that work with your
+system:
+
+include::{libbeat-dir}/tab-widgets/install-widget.asciidoc[]
+
+[float]
+[[other-installation-options]]
+==== Other installation options
+
+* <<setup-repositories,APT or YUM>>
+* https://www.elastic.co/downloads/beats/{beatname_lc}[Download page] 
+* <<running-on-docker,Docker>>
+* <<running-on-kubernetes,Kubernetes>> 
+* <<running-on-cloudfoundry,Cloud Foundry>>
+
+[float]
+[[set-connection]]
+=== Step 2: Connect to the {stack}
+
+include::{libbeat-dir}/shared/connecting-to-es.asciidoc[]
+
+[float]
+[[enable-modules]]
+=== Step 3: Enable and configure metrics collection modules
+
+{beatname_uc} uses modules to collect metrics. Each module defines the basic
+logic for collecting data from a specific service, such as Redis or MySQL. A
+module consists of metricsets that fetch and structure the data. Read
+<<how-{beatname_lc}-works>> to learn more.
+
+. Identify the modules you need to enable. To see the list of available
+<<metricbeat-modules,modules>>, run:
++
+--
+include::{libbeat-dir}/tab-widgets/list-modules-widget.asciidoc[]
+--
+
+. From the installation directory, enable one or more modules. If you accept the
+default configuration without enabling additional modules, {beatname_uc}
+collects system metrics only.
++
+The following command enables the `apache` and `mysql` configs in the
+`modules.d` directory:
++
+--
+include::{libbeat-dir}/tab-widgets/enable-modules-widget.asciidoc[]
+--
++
+See the <<modules-command>> to learn more about this command. If you are using a
+Docker image, see <<running-on-docker>>.
+
+. In the module configs under `modules.d`, change the module settings to match
+your environment. See <<module-config-options>> for more about available
+settings.
+
+include::{libbeat-dir}/shared/config-check.asciidoc[]
+
+[float]
+[[setup-assets]]
+=== Step 4: Set up assets
+
+{beatname_uc} comes with predefined assets for parsing, indexing, and
+visualizing your data. To load these assets:
+
+. Make sure the user specified in +{beatname_lc}.yml+ is
+<<privileges-to-setup-beats,authorized to set up {beatname_uc}>>.
+
+. From the installation directory, run:
++
+--
+include::{libbeat-dir}/tab-widgets/setup-widget.asciidoc[]
+--
++
+`-e` is optional and sends output to standard error instead of syslog.
+
+This step loads the recommended index template for writing to {es}
```

Comment:
I was wondering if it is a good approach to put a link to the index templates here https://www.elastic.co/guide/en/elasticsearch/reference/master/indices-templates.html for users who are unaware of them. Ignore this comment if not :smile:

Reply:
Good idea! I'll do that.